src/config.ts

```diff
@@ -37,18 +36,6 @@ const configPaths: { [domain: string]: string } = {
   project: "./skygear.yaml",
 };
 
-function migrate(configObject: Dictionary<any>) {
```

Comment:
I think we can keep this?

Reply:
If we want to help the developer to migrate their old skygear.yaml to a new one, it is better to send the old one to the controller to perform the migration.

Reply:
I think `skygear.yaml` is a structure that exists entirely within `skycli`, and `skycli` translates `skygear.yaml` into appropriate controller API arguments?

Reply:
Should the create-deployment endpoint supports versioning as well? A particular copy of skycli uses a fixed version. When the developer wants to upgrade, they install a newer skycli. skycli has a separate command to migrate `skygear.yaml` locally.

Reply:
I suppose so... that is we are going to version the 'deployment' resource.

Reply:
I will remove this function in this PR. In a later PR for https://github.com/SkygearIO/skycli/issues/185 I am going to add it back.